src/plugin.js

```diff
@@ -231,18 +231,45 @@ const onPlayerReady = (player) => {
  * to you; if not, remove the wait for "ready"!
  *
  * @function eme
+ * @param    {Object} [player]
+ *           A playerobject.
  * @param    {Object} [options={}]
  *           An object of options left to the plugin author to define.
  */
-const eme = function(options = {}) {
-  this.eme.options = options;
+const eme = function(player, options = {}) {
+  const initializeMediaKeys = (emeOptions = {}) => {
```

Comment:
It might be easier to read (and potentially test) if we make this a top level function.